### DISCUSSION TITLE
Underway Replenishment: A Call to arms

### DISCUSSION THREAD

**Discussion by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#3558] [General]
All,

I have recently noticed that many of the ships in the game have no underway replenishment or refuelling capability. even those of big, capable expeditionary navies like the UK or France. The issue is of such a scale that none of the vessels that went to the Falklands in 1982 can do so in Command, because they can't refuel on the way. Creating a comprehensive list of every missing fuel hose will require a huge amount of research and time, significantly more than I could reasonably put in myself. I would therefore like to call on everyone in the community to try to identify ships that are missing UNREP points, each covering your own area of interest or expertise.

Devs, how would you like this information to be organised? You could start an RFI thread like for datalinks, or we could do individual issues on a per ship/class basis, or it could be run through a big Google sheet with communal editing. I would estimate that at least 400 DB3k DBIDs are going to be involved here, so it's somewhat beyond the usual scope of requests.

**Comment by claudejdev** [comment]
@PygmalionOfCyprus @Jkryos 
is it something you guys think is needed?
I believe we do have the literature material to do this (as you now know it @FrangibleCover ) and we would be able to produce a sourced request for an update.
Cheers

**Comment by FrangibleCover** [comment]
This is less of an issue given that, from experimentation, ships appear to be able to take fuel without having the DB tags to do so. It's a strange situation but possibly not one that needs fixed in the short term. Of course, any missing replen _out_ capabilities need to be added, but I've not seen too many of them.
